documentation/synthetic_tokens/known_issues.md

```diff
@@ -35,6 +35,13 @@ However, this makes the role of token sponsor capital inefficient for 2 reasons:
 1. If a token sponsor wishes to create new tokens collateralized to a level below the GCR, they must first collateralize their position above the GCR and go through a “slow” withdrawal to withdraw their excess collateral.
 * Conclusion: Capital efficiency is not a primary concern at this time. 
 
+### Blocking new token sponsors by front running the GCR
+* Scenario: If two callers create positions in the same block it’s possible that one callers `create` function will revert if the other caller creates a position with enough collateral to push the GCR above the first callers CR. This occurs because all new position's CR  must be greater than or equal to the GCR. 
```

Comment:
```suggestion
* Scenario: If two callers create positions in the same block it’s possible that one caller's `create` function will revert if the other caller creates a position with enough collateral to push the GCR above the first caller's CR. This occurs because all new positions' CR must be greater than or equal to the GCR. 
```